docs/customization/backend.rst

```diff
@@ -4,9 +4,9 @@ Working with Python Code
 Managing Dependencies
 ---------------------
 
-To guarantee repeatable installations, all project dependencies are managed using `Pipenv <https://pipenv.readthedocs.io/en/latest/>`_.
-Project's direct dependencies are listed in ``Pipfile`` and running :code:`pipenv lock` would generate ``Pipfile.lock`` that has all versions pinned.
+To guarantee repeatable installations, all project dependencies are managed using `Poetry <https://poetry.eustace.io/docs/>`_.
+Project's direct dependencies are listed in ``pyproject.toml`` and running :code:`poetry lock` would generate ``poetry.lock`` that has all versions pinned.
 
-For users who are not using Pipenv, ``requirements.txt`` and ``requirements_dev.txt`` are also provided. They should be updated by :code:`pipenv lock -r > requirements.txt` and :code:`pipenv lock -r --dev > requirements_dev.txt`, if any dependencies are changed in ``Pipfile``.  
+For users who are not using Poetry, ``requirements.txt`` and ``requirements_dev.txt`` are also provided. They should be updated by :code:`poetry export --dev -f requirements.txt; mv requirements.txt requirements_dev.txt` and :code:`poetry export -f requirements.txt`, if any dependencies are added/updated. Additionally `--without-hashes` can be provided if hashes are not needed in requirement files.
```

Comment:
Should we put the ```:code:`poetry export -f requirements.txt```` first?

Reply:
See https://github.com/mirumee/saleor/pull/3894#discussion_r271592387. If this was first then exporting dev dependencies would overwrite those.

Reply:
OK. Maybe we could switch them in ```For users who are not using Poetry, ``requirements_dev.txt`` and ``requirements.txt`` are also provided.```